tasks.py

```diff
@@ -85,8 +98,8 @@ def catch_up(ctx):
 def new_db(ctx):
     """Create a new database with fake data"""
     print("* Reset the database")
-    ctx.run("dropdb --if-exists -h localhost pulse -U postgres")
-    ctx.run("createdb -h localhost pulse -U postgres")
+    ctx.run("dropdb --if-exists -h localhost -U postgres pulse")
```

Comment:
got errors on Windows due to this ordering, which is weird, but also an easy fix